doc/backing-store-crd.md

```diff
@@ -17,11 +18,23 @@ Adding support for a new type of backing-store is rather easy as it requires jus
 
 # Reconcile
 
-#### OBC type
+#### AWS-S3 type
 
-The operator will create a claim and the appropriate provisioner will create a new bucket or connect to existing one depending on the obc options. Once the claim is ready its details will be used to configure a cloud resource in NooBaa.
+Create a cloud storage agent within the NooBaa brain and use S3 API for storing encrypted chunks of data in the AWS cloud.
```

Comment:
Change terminology to match the noobaa UI - "Create a cloud resource within the NooBaa brain ..."

Reply:
This comment applies to all the types below